grype/presenter/cyclonedx/vulnerability.go

```diff
@@ -36,17 +36,17 @@ type Source struct {
 
 // Rating has information about the intensity of a vulnerability
 type Rating struct {
-	Score    Score  `xml:"v:score"`
+	Score    Score  `xml:"v:score,omitempty"`
```

Comment:
Since the type of the `Score` field is not a pointer, this struct tag essentially has no effect. For it to be truly optional we would need to change from `Score` to `*Score`.

Reply:
In my testing it did have an effect once the fields inside Score had omitempty, but I can double check that. Or would you prefer the pointer either way to be more explicit that the struct might not be present?